ELFSage/Test/TestRunner.lean

```diff
@@ -9,29 +9,28 @@ private def EXAMPLE_TEST_NAME: System.FilePath := System.mkFilePath [ "true" ]
 
 def runSingleTest (elfFile: System.FilePath) (expectedOutputFile: System.FilePath)
     (testOutputFile: System.FilePath): IO UInt32 := do
+  -- TODO: Clean this up. This file is being written so that, if it did not previously exist,
+  -- deleting it will not error.
+  IO.FS.writeFile testOutputFile ""
```

Comment:
@gleachkr do you know how to do this? It might be as simple as suppressing the writeFile error if it is thrown

Reply:
Yep! Error handling for IO is via [this typeclass](https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html#MonadExceptOf). So you can do:

```
try IO.FS.removeFile testOutputFile
catch e => pure ()
```
or, since in this case you don't really want to do anything with the exception, 
```
IO.FS.removeFile testOutputFile <|> pure ()
```
which uses [OrElse](https://leanprover-community.github.io/mathlib4_docs/Init/Prelude.html#OrElse)

Reply:
Great. I'll clean this up the next time I'm in the code.